content/documentation/pages/5-feature-guides/2-batch/4-monitoring.md

```diff
@@ -0,0 +1,170 @@
+--spring.cloud.dataflow.applicationProperties.task.management.metrics.export.influx.uri=http://localhost:8086
+--spring.cloud.dataflow.grafana-info.url=http://localhost:3000
+```
+
+Instead of having to install them manually, for a quick bootstrap, Spring Cloud Data Flow provides a [Docker Compose Influx](https://github.com/spring-cloud/spring-cloud-dataflow/blob/master/spring-cloud-dataflow-server/docker-compose-influxdb.yml) file which bring up Spring Cloud Data Flow, Skipper, Apache Kafka, Influx, and prebuilt dashboards for Grafana. Instructions below leverage this approach.
+
+[[tip | Upgrade to latest version of Docker ]]
+| We recommended that you upgrade to the [latest version](https://docs.docker.com/compose/install/) of Docker before running the `docker-compose` command. We have tested with Docker Engine version `18.09.2`.
+
+- Downloading the Docker Compose Influx file
+
+To download the Spring Cloud Data Flow Server Docker Compose file, run the following command:
+
+```bash
+wget https://raw.githubusercontent.com/spring-cloud/spring-cloud-dataflow/v%dataflow-version%/spring-cloud-dataflow-server/docker-compose-influxdb.yml
+```
+
+- Starting Docker Compose
+
+In the directory where you downloaded `docker-compose-influxdb.yml`, start the system, by running the following commands:
+
+```bash
+export DATAFLOW_VERSION=%dataflow-version%
+export SKIPPER_VERSION=%skipper-version%
+docker-compose -f ./docker-compose-influxdb.yml up
+```
+
+Now that docker compose is up, you can access the [Spring Cloud Data Flow Dashboard](http://localhost:9393/dashboard). Also you can reach the Grafana dashboard at http://localhost:3000 using the user: admin, password: admin credentials.
+
+Now you can deploy a custom Task application (`task-demo-metrics`) and define two tasks (`task1` and `task2`):
+
+```bash
+dataflow:>app register --name myTask --type task --uri https://github.com/tzolov/task-demo-metrics/raw/master/apps/task-demo-metrics-0.0.1-SNAPSHOT.jar
+
+dataflow:>task create --name task1 --definition "myTask"
+dataflow:>task create --name task2 --definition "myTask"
+```
+
+Launch the tasks several times:
+
+```bash
+dataflow:>task launch --name task1
+dataflow:>task launch --name task2
+```
+
+In the [DataFlow task execution UI](http://localhost:9393/dashboard/#/tasks/executions) you should see list like this:
+![SCDF Task Execution](images/SCDF-metrics-task-execution-view.png)
+
+You should see dashboards similar to those shown in the following image:
+
+![SCDF Task Grafana InfluxDB](images/SCDF-metrics-grafana-task.png)
+
+## Cloud Foundry
+
+This section describes how to set up InfluxDB for Cloud Foundry.
+
+### InfluxDB
+
+You can follow the general [Manifest based installation on Cloud Foundry](http://localhost:8000/docs/installation/cloudfoundry/cf-cli/#manifest-based-installation-on-cloud-foundry) instructions for installing `Skipper` and `DataFlow` on Cloud Foundry.
```

Comment:
A bunch of localhost links here. It should instead point to the relative path to where the markdowns reside.

Reply:
ah, good spot